src/quick-tour/reduce-ordering-errors.md

```diff
@@ -3,23 +3,22 @@ conditions: Default.B2B Only
 title: Reduce Ordering Errors
 ---
 
-[Streamlined Order Process]({{ site.baseurl }}{% link sales/point-of-purchase.md %})
-:  [Quick Order]({{ site.baseurl }}{% link sales/quick-order.md %})
-reduces the process to several clicks. Magento offers multiple ways to configure the [shopping cart]({{ site.baseurl }}{% link sales/cart-configuration.md %}) and [checkout process]({{ site.baseurl }}{% link sales/checkout-configuration.md %}), to eliminate manual data entry, and reduce costly ordering errors.
+Streamlined Order Process
+:  [Quick Order]({{ site.baseurl }}{% link sales/quick-order.md %}) reduces the process to several clicks. Magento offers multiple ways to configure the [shopping cart]({{ site.baseurl }}{% link sales/cart-configuration.md %}) and [checkout process]({{ site.baseurl }}{% link sales/checkout-configuration.md %}), to eliminate manual data entry and reduce costly ordering errors.
 
-[Assisted Sales]({{ site.baseurl }}{% link sales/shopping-assistance.md %})
-:  Assign a dedicated sales rep to each company account, who can access the customer’s shopping cart in real time, and offer personal assistance over the phone.
+Assisted Sales
+:  Assign a [dedicated sales rep]({{ site.baseurl }}{% link sales/shopping-assistance.md %}) to each company account, who can access the customer’s shopping cart in real time, and offer personal assistance over the phone.
```

Comment:
agent tasks need to come before account.

Assign a dedicated sales rep, who can access the customer’s shopping cart in real time and offer personal assistance over the phone, to each company account.